test/library/packages/Sort/RadixSort/radixMSBModule.chpl

```diff
@@ -0,0 +1,83 @@
+ use BitOps;
+ use Sort;
+ use Random;
+ config param bucketBits=8; 
+
+ module MSBmodule {
+
+   proc radixSortMSB(array:[?dom] int) {
+     var auxArray:[1..array.size] int;
+     var place = findPlace(array);
+     __radixSortMSB(array,place,dom.low,dom.high,auxArray);
+   }
+  
+   private proc __radixSortMSB(array:[] int, place: int, startIndex: int, endIndex: int, auxArray:[] int) {
+
+     param numBuckets = (1 << bucketBits);
+     var counts:[1..numBuckets+1] int;  
+     var currentDigit: int; 
+     var countIndex: int;
+     var auxIndex: int;
+   
+     //base case, exit condition
```

Comment:
In the future this should call a comparison sorter for < 100 or so elements. You could do that now if you like - it should significantly improve performance.

Reply:
If you decide to put this off, make a `// TODO: call comparison sorter for small numbers of elements`.